Bottom: 86ed9f8eb07f0149429219c4be2e49a397e16fe8
Top:    b4cf7ded0b8a1d3636edabb955dac5d7f7da2716
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-15 22:19:27 +0000

Refresh of tmp-changes

---

diff --git a/src/polygon1.c b/src/polygon1.c
index e9f2efc..0775646 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1817,7 +1817,6 @@ M_B_AREA_Collect (jmp_buf * e, POLYAREA * bfst, POLYAREA ** contours,
 	       * NB: There Could be grief if the JUMP rule is inconsistent in
 	       *     its interpretation from each side of the vertex.
 	       */
-#if 1
             switch (action)
               {
               case PBO_UNITE:
@@ -1833,7 +1832,6 @@ M_B_AREA_Collect (jmp_buf * e, POLYAREA * bfst, POLYAREA ** contours,
                 Collect ('B', e, *cur, contours, holes, SubS_Rule, SubJ_Rule);
                 break;
               }
-#endif
 	    }
           else if ((*cur)->Flags.status == INSIDE)
 	    switch (action)
